scripts/setup/authentication.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import requests
+from typing import List, Optional
+
+from fides.api.ops.api.v1.scope_registry import SCOPE_REGISTRY
+from fides.api.ops.api.v1 import urn_registry as urls
+
+import constants
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_access_token(
+    client_id: str,
+    client_secret: str,
+) -> str:
+    """
+    Authorize with fidesops via OAuth.
```

Comment:
I don't think we're delineating between ctl/ops oauth anymore, they all use the same user endpoints/authentication